ui/app/AppLayouts/Profile/panels/CommunitiesListPanel.qml

```diff
@@ -43,17 +44,14 @@ StatusListView {
 
         components: [
             StatusFlatButton {
+                anchors.verticalCenter: parent.verticalCenter
                 objectName: "CommunitiesListPanel_leaveCommunityPopupButton"
                 size: StatusBaseButton.Size.Small
                 type: StatusBaseButton.Type.Danger
                 borderColor: "transparent"
-                text: qsTr("Leave Community")
-                onClicked: {
-                    Global.openPopup(leaveCommunityPopup, {
-                                         community: model.name,
-                                         communityId: model.id
-                                     })
-                }
+                enabled: !model.amISectionAdmin
+                text: model.spectated ? qsTr("Close Community") : qsTr("Leave Community")
+                onClicked: model.spectated ? root.closeCommunityClicked(model.id) : root.leaveCommunityClicked(model.name, model.id, model.outroMessage)
```

Comment:
I wonder if instead of doing the condition here, we could do it in the parent component, so that here, we just call the `leave` signal (one less signal)

Reply:
Ah ok, no problem to leave it as is then